src/Components/Actor/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import './style.css';
+
+const Actor = (props) => {
+  return (
+    <>
+      <div className="actorTable">
+        <div className="name"> {props.name} </div>
```

Comment:
jenom dej bacha, že všechny tyhle názvy class jsou globální, tj nic ti nebrání použít stejný classname i v jiné komponentě. Takže většinou se aspoň dělá to, že si ty classy pojmenuješ tak, aby ses vyznala aby byly per komponenta, třeba "`actor_name`", "`actor_as`", ...